iamclient/iamclient_test.go

```diff
@@ -416,3 +424,13 @@ func randomString() string {
 
 	return string(secret)
 }
+
+func instanceIdentToPB(ident cloudprotocol.InstanceIdent) *pb.InstanceIdent {
```

Comment:
`pbconvert.InstanceIdentToPB` maybe use this?

Reply:
the same